src/pydiverse/transform/lazy/sql_table/sql_table.py

```diff
@@ -617,10 +628,16 @@ def over_value(*args, **kwargs):
                 )
 
                 v = value(*args, **kwargs)
-                return v.over(
+                post_process = None
+                if isinstance(v, tuple):
+                    v, post_process = v
+                res = v.over(
                     partition_by=pb,
                     order_by=ob,
                 )
+                if post_process:
+                    res = post_process(res)
+                return res
```

Comment:
Can you add test / operator that uses this post processing step? It seems like this is currently just dead code.